modules/imgcodecs/src/grfmt_png.hpp

```diff
@@ -86,12 +61,40 @@ class PngEncoder CV_FINAL : public BaseImageEncoder
 
     bool  isFormatSupported( int depth ) const CV_OVERRIDE;
     bool  write( const Mat& img, const std::vector<int>& params ) CV_OVERRIDE;
+    bool  writemulti(const std::vector<Mat>& img_vec, const std::vector<int>& params) CV_OVERRIDE;
```

Comment:
It's ok to keep this API for now, for debugging, but I'd like to see a proper animation API, maybe in private, that starts taking into account the frame durations.